src/styles/tokens.ts

```diff
@@ -0,0 +1 @@
+export const CURVE_DAO_TOKEN_ADDRESS = '0xD533a949740bb3306d119CC777fa900bA034cd52';
```

Comment:
not sure it make sense for this to be under the styles directory. maybe the constant directory?
If this is one off, I think its also ok to just have the constant be in [TokenSelectMenu.tsx](https://github.com/dydxprotocol/v4-web/pull/506/files#diff-b4930b6454894540bd1456d1f733f23c0a32ceffb1bc3b2340a4ff5828aeb9d3)

Reply:
file search took me to `/styles/constants` instead of `/constants` 😆